modules/infra/cli/port.c

```diff
@@ -24,12 +25,28 @@ static void port_show(const struct gr_api_client *, const struct gr_iface *iface
 	printf("n_txq: %u\n", port->n_txq);
 	printf("rxq_size: %u\n", port->rxq_size);
 	printf("txq_size: %u\n", port->txq_size);
+
+	if (iface->mode == GR_IFACE_MODE_L1_XC) {
+		if (iface_from_id(c, iface->domain_id, &peer) == 0)
+			printf("xc peer: %s\n", peer.name);
+		else
+			printf("xc peer: unknown\n");
+	}
 }
 
 static void
-port_list_info(const struct gr_api_client *, const struct gr_iface *iface, char *buf, size_t len) {
+port_list_info(const struct gr_api_client *c, const struct gr_iface *iface, char *buf, size_t len) {
 	const struct gr_iface_info_port *port = (const struct gr_iface_info_port *)iface->info;
-	snprintf(buf, len, "devargs=%s mac=" ETH_F, port->devargs, &port->mac);
+	struct gr_iface peer;
+	size_t s;
+
+	s = snprintf(buf, len, "devargs=%s mac=" ETH_F, port->devargs, &port->mac);
+	if (iface->mode == GR_IFACE_MODE_L1_XC) {
+		if (iface_from_id(c, iface->domain_id, &peer) == 0)
+			snprintf(&buf[s], len - s, " xc peer -> %s", peer.name);
+		else
+			snprintf(&buf[s], len - s, " xc peer -> unknown");
```

Comment:
Follow the same style as the rest of the attributes:

"xc_peer=$peer_name" or if iface_from_id fails, "xc_peer=$domain_id"

Can you use SAFE_BUF here?